### PR TITLE
change /usr/bin/bash to /bin/bash in the edgemesh_init image

### PR DESCRIPTION
**Pull request by cwl233**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change

/kind bug

> /kind cleanup
> /kind design
> /kind documentation
> /kind test
> /kind failing-test
> /kind feature

**What this PR does / why we need it**:
I tried to build the edgemesh_init image by myself, the container can not be run because the bash is in the directory /bin/ but not /usr/bin/.
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: edgemesh/tools/initContainer/script/edgemesh-iptables.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/bash
+#!/bin/bash
```
Comment (fisherxu) [inline edgemesh/tools/initContainer/script/edgemesh-iptables.sh]: `#!/usr/bin/env bash`  seems better, which searches PATH for bash, and bash is not always in /bin, particularly on non-Linux systems. Thought? Xref:  https://stackoverflow.com/questions/21612980/why-is-usr-bin-env-bash-superior-to-bin-bash

Comment (cwl233) [reply edgemesh/tools/initContainer/script/edgemesh-iptables.sh]: yes, "#!/usr/bin/env bash" is better

**Pull request merged**